wrangler-transform/src/main/java/io/cdap/wrangler/Wrangler.java

```diff
@@ -524,6 +531,34 @@ private void validatePrecondition(String precondition, Boolean isConditionSQL, F
   }
 
   private void validateSQLModeDirectives(FailureCollector collector) {
```

Comment:
Can change the function to take a list of directives and check whether they are sql supported. It seems like the function performs too many different operations